core/src/main/scripts/importer/validateData.py

```diff
@@ -2135,12 +2142,22 @@ def checkHeader(self, cols):
                     if attr_property == 'is_patient_attribute':
                         expected_level = self.PREDEFINED_ATTRIBUTES[col_name][attr_property]
                         if self.PROP_IS_PATIENT_ATTRIBUTE != expected_level:
-                            self.logger.error(
-                                'Attribute must to be a %s-level attribute',
-                                {'0': 'sample', '1': 'patient'}[expected_level],
-                                extra={'line_number': self.line_number,
-                                       'column_number': col_index + 1,
-                                       'cause': col_name})
+                            if col_name in self.WARNING_ATTRIBUTES:
+                                self.logger.warning(
+                                    'Clinical Data Dictionary defines this as a %s-level attribute. '
+                                    'See GET /api/ at http://oncotree.mskcc.org/cdd/swagger-ui.html#/',
```

Comment:
I'm not sure what that ‘Clinical Data Dictionary’ is for, so how should I respond when I see this warning? What breaks if I completely ignore it?

Reply:
We could add an explanation of this in File-Formats. If I understand correctly, the CDD is the format used by MSKCC to normalize clinical data, and should be followed for the study to be compared to other studies. Not following this format is okay for non-core attributes, but then the users should know that the clinical data cannot be compared.

Reply:
Is it possible to make the warning message itself let the user know that, so that they can quickly decide whether they want to accept this potential problem or refer to external docs to fix it?

Reply:
@fedde-s Good point! I changed the warning and added the CDD to file formats.

Reply:
That resolves the potential issue I saw 👍